cloudify_agent/resources/script/linux-download.sh.template

```diff
@@ -2,16 +2,21 @@
 
 # Download and execute the script that will take care of the agent installation
 
+add_ssl_cert()
+{
+    # Create all the directories in the path to the cert file
+    mkdir -p $(dirname {{ ssl_cert_path }})
+    echo "{{ ssl_cert_content }}" > {{ ssl_cert_path }}
+}
+
 download()
 {
     SCRIPT_NAME=$1
 
-    # Downloading with '--insecure' flags, because at this point we don't have
-    # the SSL certificates
     if command -v wget > /dev/null 2>&1; then
-        wget {{ link }} -O ${SCRIPT_NAME} --no-check-certificate
+        wget {{ link }} -O ${SCRIPT_NAME} --ca-certificate {{ ssl_cert_path }}
```

Comment:
Should be fine, really - it's never been quoted in other places (like `linux.sh.template`)